soc/espressif/common/empty.ld

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2025 Espressif Systems (Shanghai) Co., Ltd.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Empty linker script to allow MCUboot Espressif Port to link in its
+ * own way when building using sysbuild.
+ */
```

Comment:
I think EP is missing the memory allocations for the bootloader images that are being calculated in the `memory.h`. The current layout in the `boot/espressif/port/<soc>/bootloader.ld` could interfere with the images with higher SRAM consumption. Ex. ESP32-S3 `iram_loader.text` is normally at the `0x403c6400`, while the EP integration ld sets this to the `0x403ba000`, which is ~50kB less.